modules/net-interconnect-attachment-direct/README.md

```diff
@@ -0,0 +1,131 @@
+﻿# Direct Interconnect VLAN Attachment and router
+
+
+This module allows creation of a VLAN attachment for Direct Interconnect and router.
+
+## Examples
+
+### Direct Interconnect  VLAN attachment using default parameters for bgp session and router
+
+module "vlan-attachment-1" {
+  source = "./modules/net-interconnect-attachment-direct"
+  project_id   = "dedicated-ic-5-8492"
+  region       = "us-west2"
+  network_name = "myvpc"
+  vlan_attachment_name = "vlan-604-x"
+   interconnect  = "https://www.googleapis.com/compute/v1/projects/cso-lab-management/global/interconnects/cso-lab-interconnect-1"
```

Comment:
plz don't use real resources in examples, use something like `projects/my-project/global/interconnects/my-ic-1`